### ISSUE TITLE
Update to Laravel 5.6

### ISSUE THREAD

**Issue by gbgelado** [fruitcake/laravel-cors#287]
```
- Installation request for barryvdh/laravel-cors ^0.9.2 -> satisfiable by barryvdh/laravel-cors[v0.9.2, v0.9.3].
```

**Comment by gghughunishvili** [comment]
Try this:
`composer require barryvdh/laravel-cors`

This will install latest version..

And run update again. After that, it should work.
Answer can be found from closed issue #286

**Comment by GrahamCampbell** [comment]
I think this issue can be closed @barryvdh?
